src/commissaire_service/storage/__init__.py

```diff
@@ -107,19 +110,19 @@ def register_store_handler(self, config):
         self._manager.register_store_handler(
             handler_type, config, *matched_types)
 
-    def _build_model(self, model_type_name, model_kwargs):
+    def _build_model(self, model_type_name, model_json_data):
```

Comment:
I'm seeing the following when using ```commissaire-storage-service```:
```
Exception raised during method call:
Traceback (most recent call last):
  File ".../commissaire-service/src/commissaire_service/service/__init__.py", line 216, in _wrap_on_message
    result = method(message, *body['params'])
  File ".../commissaire-service/src/commissaire_service/storage/__init__.py", line 146, in on_save
    model_instance = self._build_model(model_type_name, model_json_data)
  File ".../commissaire-service/src/commissaire_service/storage/__init__.py", line 125, in _build_model
    return model_type.new(**json.loads(model_json_data))
  File "/usr/lib64/python3.5/json/__init__.py", line 312, in loads
    s.__class__.__name__))
TypeError: the JSON object must be str, not 'dict'
```

With current HEAD (52da60f8c844336ad97810a1f6f4a1868e63e532) the same requests work.